training/distributed_training/pytorch/model_parallel/gpt2/submit-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>Grammar - pluralize - <code>the following files/folders</code></p><p> </p><p>Be consistent with use of period or no period at end of list lines.</p><p> </p><p>Prerequisites - use the default SageMaker S3 bucket. No need to create a new one. This is what your code is already doing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
User needs to create a S3 bucket for their own data. The notebook is using the default SageMaker S3 bucket with CI data only.